ex7.6_NumPy_compute_x_y.py

```diff
@@ -0,0 +1,17 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+def compute_x_y_coordinates():
```

Comment:
Use type hints and return types -docum
https://www.geeksforgeeks.org/duck-typing-in-python/
https://stackoverflow.com/questions/125367/dynamic-type-languages-versus-static-type-languages
https://docs.python.org/3/library/typing.html
https://realpython.com/lessons/pros-and-cons-type-hints/